image-puller.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs').promises;
```

Comment:
This file shouldn't live in the root.  I think we should probably have it get integrated with our Gatsby build process via one of the lifecycle hooks, maybe [onPreInit](https://www.gatsbyjs.org/docs/node-apis/#onPreInit).  I've not done this before, but from my readings, you can write code that hooks into different parts of a build, and we could call into this code in our build steps within Gatsby.

Reply:
Or maybe [onPreBuild](https://www.gatsbyjs.org/docs/node-apis/#onPreBuild).